nodes/src/nodes/agent_langchain/langchain.py

```
@@ -67,7 +67,7 @@ def _llm_type(self) -> str:
             def _identifying_params(self) -> Dict[str, Any]:
                 return {'framework': 'rocketride', 'adapter': 'tool_calling_json'}
 
-            def bind_tools(self, tools: Any, **kwargs: Any) -> "RocketRideToolCallingChatModel":
+            def bind_tools(self, tools: Any, **kwargs: Any) -> 'RocketRideToolCallingChatModel':
```

Ruff (0.15.7)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `tools`

(ANN401)